src/main/java/racingcar/controller/RaceGameController.java

```diff
@@ -0,0 +1,29 @@
+package racingcar.controller;
+
+import java.util.List;
+import racingcar.dto.InputDTO;
+import racingcar.model.RaceJudge;
+import racingcar.model.RacingCar;
+import racingcar.view.InputView;
+import racingcar.view.RaceWinnerView;
+
+public class RaceGameController {
+    private final InputView inputView;
+    private final RandomRaceController randomRaceController;
+    private final RaceWinnerView raceWinnerView;
+
+    public RaceGameController() {
+        this.inputView = new InputView();
+        this.randomRaceController = new RandomRaceController();
+        this.raceWinnerView = new RaceWinnerView();
+    }
+
+    public void gameStart() {
+        InputDTO inputDTO = inputView.getInput();
+        Long tryNum = inputDTO.tryNums();
+        List<RacingCar> racingCars = inputDTO.to();
+        List<RacingCar> participantCars = randomRaceController.start(racingCars, tryNum);
```

Comment:
컨트롤러에서 컨트롤러를 직접 호출하셨네여!
이렇게 작성한 이유가 있을까요..? 순수한 질문입니돠...🐹

아참! 컨트롤러로 넘길거면 인수로 inputDTO를 그냥 넘기는 것은 어떨까요?
```suggestion
        List<RacingCar> participantCars = randomRaceController.start(inputDTO);
```

Reply:
헉 그점이 더 좋아보이네요. 이런 목적으로 DTO를 생성해두고 이용하지 않았던 것 같습니다...